app/controllers/search_controller.rb

```diff
@@ -167,49 +158,33 @@ def feed_content
           term: feed_params[:search_fields],
         )
       elsif class_name.User?
-        if FeatureFlag.enabled?(:search_2_users)
-          Search::Postgres::User.search_documents(
-            term: feed_params[:search_fields],
-            page: feed_params[:page],
-            per_page: feed_params[:per_page],
-            sort_by: feed_params[:sort_by] == "published_at" ? :created_at : nil,
-            sort_direction: feed_params[:sort_direction],
-          )
-        else
-          user_search
-        end
-      elsif class_name.Article? && FeatureFlag.enabled?(:search_2_articles, current_user)
+        Search::Postgres::User.search_documents(
+          term: feed_params[:search_fields],
+          page: feed_params[:page],
+          per_page: feed_params[:per_page],
+          sort_by: feed_params[:sort_by] == "published_at" ? :created_at : nil,
+          sort_direction: feed_params[:sort_direction],
```

Comment:
You may notice there are many opportunities to refactor feed_content. A lot of the searches use the same params, for example. Our thinking here is we're going to punt on this because the goal is to get Elasticsearch out ASAP. We'd also like to explore breaking these searches into their own endpoints instead of this large switch operator all under feed_content.